api-tests/src/clients/ticf-management-api.ts

```diff
@@ -1,41 +1,41 @@
 import config from "../config/config.js";
 import { DataTable } from "@cucumber/cucumber";
+import { TicfManagementParameters } from "../types/ticf-management-api.js";
+import { getRandomString } from "../utils/random-string-generator.js";
 
 export const parseTableForTicfManagementParameters = (table: DataTable) => {
   const rowsHash = table.rowsHash();
   const responseDelay = parseInt(rowsHash.responseDelay) || 0;
   const cis = rowsHash.cis && rowsHash.cis.split(",");
 
   return {
-    cis: cis || undefined,
+    ci: cis || undefined,
     responseDelay,
-    type: rowsHash.type,
-    txn: rowsHash.txn || undefined,
+    type: rowsHash.type || "RiskAssessment",
+    txn: rowsHash.txn === "" ? undefined : getRandomString(16),
     statusCode: rowsHash.statusCode,
   };
 };
 
 export const postUserToTicfManagementApi = async (
   userId: string,
-  cis: string[] | undefined,
-  type: string,
-  responseDelay: number,
-  txn: string | undefined,
-  statusCode: string | undefined,
+  parsedTicfManagementParameters: {
+    statusCode: string;
+  } & TicfManagementParameters,
 ) => {
+  const { statusCode } = parsedTicfManagementParameters;
```

Comment:
This is a bit awkward isn't it :(

I'm not sure what the rationale behind making `statusCode` a url param (rather than body param like everything else)... Perhaps a tech debt ticket for us.